src/getRandomNumber.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const randomNumberOfFourDigits = Math.floor(Math.random() * 9000) + 1000;
+
+module.exports = randomNumberOfFourDigits;
```

Comment:
```suggestion
module.exports = randomNumberOfFourDigits.toString();
```

You can export number as string here to use it in your main module.